docs/layers/introduction.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Introduction
+sidebar_label: Introduction
+slug: /layers
+---
+
+# What are Layers?
+Layers are a way of performing experiments with out requring continued updates to your codebase. They are a 'write once' solution to experimentation. Layers work with Experiments to run tests on product features, and allow shared parameters between Experiments.
+
+## When to use Layers
+Similar to our Experiments product, Layers are ideal for running A/B or A/B/n experiments. There differentiator is that you can avoid rewritting code between experiment runs.
+
+
+## Layers vs Experiments 
```

Comment:
Maybe "Layers vs Standalone Experiments" or something? Just want to make it clear that you still use experiments. Maybe it's fine though